Corona-Warn-App/src/main/res/layout/home_reenable_risk_card_layout.xml

```diff
@@ -0,0 +1,106 @@
+            android:text="@string/submission_test_result_card_positive_title"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/reenable_risk_card_title" />
+
+        <ImageView
+            android:id="@+id/reenable_risk_card_positive_icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:background="@drawable/ic_test_result_illustration_positive_card"
+            android:importantForAccessibility="no"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/reenable_risk_card_positive_title"
+            tools:src="@drawable/ic_test_result_illustration_positive" />
+
+        <TextView
+            android:id="@+id/reenable_risk_card_positive_body"
+            style="@style/body2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:layout_marginEnd="@dimen/spacing_small"
+            android:text="@string/submission_test_result_card_positive_body"
+            app:layout_constraintEnd_toStartOf="@id/reenable_risk_card_positive_icon"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/reenable_risk_card_positive_title" />
+
+        <TextView
+            android:id="@+id/reenable_risk_card_test_registration_date"
+            style="@style/body2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/spacing_small"
+            android:text="@{state.formatTestRegistrationText(context)}"
+            tools:text="@string/reenable_risk_card_test_registration_string"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/reenable_risk_card_positive_body" />
+
+        <View
+            android:id="@+id/reenable_risk_card_divider"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/card_divider"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:background="@color/colorHairline"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/reenable_risk_card_test_registration_date" />
+
+        <TextView
+            android:id="@+id/reenable_risk_card_description"
+            style="@style/body2"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_medium"
+            android:layout_marginEnd="@dimen/spacing_small"
+            android:text="@string/reenable_risk_card_description_text"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/reenable_risk_card_divider" />
```

Comment:
Top and bottom text have the same style (`body2`), while in the mockups the bottom text has a larger font size. Intentional change?

![Screenshot from 2021-02-08 15-40-21](https://user-images.githubusercontent.com/1439229/107234804-560d2100-6a24-11eb-9782-c29056fb3afc.png)

Reply:
Didn't catch that. It's indeed a bit bigger in Figma.

Reply:
I've increased the font size to match Figma.